scripts/test.py

```diff
@@ -179,6 +179,16 @@ def run_test(driver: Path, log_queue: queue.Queue) -> int:
         )
         return 0
 
+    # GCC Reference Output
+    gcc_result = subprocess.run(
+        [
+            "riscv64-unknown-elf-gcc",
+            "-march=rv32imfd", "-mabi=ilp32d",
+            "-o", f"{log_path}.gcc.s",
+            "-S", str(to_assemble)
```

Comment:
We need more flags, following:
`riscv64-unknown-elf-gcc -std=c90 -pedantic -ansi -O0 -march=rv32imfd -mabi=ilp32d -S [source-file.c] -o [dest-file.s]`
```suggestion
            "riscv64-unknown-elf-gcc",
            "-std=c90",
            "-pedantic",
            "-ansi",
            "-O0",
            "-march=rv32imfd",
            "-mabi=ilp32d",
            "-o", f"{log_path}.gcc.s",
            "-S", str(to_assemble)
```